### PR TITLE
chore: move from travis to github actions

### PR DESCRIPTION
**Pull request by ableuler**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on: [push]
+
+jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: rokroskar/workflow-run-cleanup-action@master
```
Comment (ableuler) [inline .github/workflows/ci.yml:9]: @rokroskar maybe you can move this one to the renku repo or somewhere in the SDSC orgM

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    if: "startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'"
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install helm
+        env:
+          HELM_URL: https://storage.googleapis.com/kubernetes-helm
+          HELM_TGZ: helm-v2.16.1-linux-amd64.tar.gz
```
Comment (pameladelgado) [inline .github/workflows/ci.yml:42]: why not 2.16.9?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rokroskar) [reply .github/workflows/ci.yml:9]: I'm not sure it's necessary but yes, could do that at some point. sdsc can also fork it.

**Comment by rokroskar** [comment]
LGTM but can you comment out https://github.com/SwissDataScienceCenter/renku-gateway/pull/264/files#diff-e9f950f17198d3d5e3122a44230a09b9R33 and https://github.com/SwissDataScienceCenter/renku-gateway/pull/264/files#diff-e9f950f17198d3d5e3122a44230a09b9R58 once just to make sure it runs? Then revert the commit and we can merge.

**Comment by rokroskar** [comment]
Also this PR should remove the travis configuration and scripts

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ableuler) [reply .github/workflows/ci.yml:42]: because I was lazy and just copy-pasted from the notebooks service... fair point :)

**Comment by ableuler** [comment]
> Also this PR should remove the travis configuration and scripts

It does - or did I miss something?

**Comment by rokroskar** [comment]
> It does - or did I miss something?

haha no I just didn't see it 👍

**Comment by ableuler** [comment]
> LGTM but can you comment out https://github.com/SwissDataScienceCenter/renku-gateway/pull/264/files#diff-e9f950f17198d3d5e3122a44230a09b9R33 and https://github.com/SwissDataScienceCenter/renku-gateway/pull/264/files#diff-e9f950f17198d3d5e3122a44230a09b9R58 once just to make sure it runs? Then revert the commit and we can merge.

Looks all good: https://github.com/SwissDataScienceCenter/renku-gateway/runs/979516734?check_suite_focus=true

**Pull request merged**